app/controllers/content/index.js

```diff
@@ -78,6 +80,15 @@ export default Ember.Controller.extend(Analytics, {
             this.get('model.dateCreated');
     }),
 
+    editButtonLabel: Ember.computed('model.provider.reviewsWorkflow', 'model.reviewsState', function () {
+        const edit_preprint = 'content.project_button.edit_preprint';
+        const edit_resubmit_preprint = 'content.project_button.edit_resubmit_preprint';
+        return (
+            this.get('model.provider.reviewsWorkflow') === PRE_MODERATION
+            && this.get('model.reviewsState') == REJECTED
```

Comment:
Why `===` for the workflow but `==` for the state?

Reply:
was a typo